engine/src/main/java/io/camunda/zeebe/process/test/engine/GrpcToLogStreamGateway.java

```diff
@@ -137,6 +155,26 @@ public void activateJobs(
     writer.writeCommandWithoutKey(jobBatchRecord, recordMetadata);
   }
 
+  @Override
+  public void streamActivatedJobs(
+      final StreamActivatedJobsRequest request,
+      final StreamObserver<ActivatedJob> responseObserver) {
+    final var jobActivationProperties = new JobActivationPropertiesImpl();
+    final var worker = wrapString(request.getWorker());
+    final var jobType = wrapString(request.getType());
+    final var serverObserver = (ServerCallStreamObserver<ActivatedJob>) responseObserver;
+    final var consumer = new GrpcJobConsumer(jobType, serverObserver);
+    jobActivationProperties
+        .setWorker(worker, 0, worker.capacity())
+        .setTimeout(request.getTimeout())
+        .setFetchVariables(request.getFetchVariableList().stream().map(StringValue::new).toList())
+        .setTenantIds(request.getTenantIdsList());
```

Comment:
:thought_balloon: Even though we don't support tenants yet here, not adding anything here would prevent ever streaming jobs (since there would be no tenant with the job stream, and the engine at least expects a default tenant).